docs/release-notes/flink-1.10.md

```diff
@@ -176,6 +182,18 @@ The container cut-off configuration options, `containerized.heap-cutoff-ratio`
 and `containerized.heap-cutoff-min`, have no effect for task executor processes
 anymore but they still have the same semantics for the JobManager process.
 
+#### RocksDB State Backend Memory Control ([FLINK-7289](https://issues.apache.org/jira/browse/FLINK-7289))
+Together with the introduction of the [new Task Executor Memory
+Model](#new-task-executor-memory-model-flink-13980), the memory consumption of the RocksDB state backend will be
```

Comment:
We are depending on #10999 here to get the valid link, right?

Reply:
yes

Reply:
Actually this is a valid link pointing to the section above. We were waiting for the link in

```
More
details and advanced configuration options can be found in the [Flink user
documentation]({{ site.baseurl }}/ops/state/large_state_tuning.html#tuning-rocksdb-memory)
```

Reply:
I see, the [tuning-rocksdb-memory](https://ci.apache.org/projects/flink/flink-docs-release-1.10/ops/state/large_state_tuning.html#tuning-rocksdb-memory) link is already valid, too.